sequencing-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -154,6 +162,26 @@ declare global {
     public GET_DESCRIPTION(): Description | undefined;
   }
 
+  // @ts-ignore : 'HardwareCommand' found in JSON Spec
+  class HardwareStem implements HardwareCommand {
```

Comment:
Since some of these properties are shared now between `HardwardStem` and `CommandStem` do you think it's worth moving them to a shared interface? I don't know if we'll be adding more types that have `METADATA` and `DESCRIPTION`s.

Reply:
I thought about it but wanted to hold off until we get the full spec implemented and then do that type of refactoring. I didn't want to open a can of worms and afraid of blowing up all the test.

Reply:
Sure as long as you're keeping it in your mind! There's probably a point where it makes sense before we finish the spec though.

Reply:
Yep, I created a ticket a while ago.